src/main/java/io/axoniq/axonserver/connector/event/EventChannel.java

```diff
@@ -387,4 +390,57 @@ default ResultStream<EventQueryResultEntry> querySnapshotEvents(String queryExpr
     default ResultStream<EventQueryResultEntry> querySnapshotEvents(String queryExpression, boolean liveStream, String contextName) {
         throw new UnsupportedOperationException();
     }
+
+    /**
+     * Opens the persistent stream identified by {@code streamId}. Fails if the stream does not exist.
+     * @param streamId the unique identification of a persistent stream
+     * @return a PersistedStream streaming events per segment
+     */
+    PersistentStream openPersistentStream(String streamId);
+
+    /**
+     * Opens the persistent stream identified by {@code streamId}. If the stream does not exist it will be
+     * created with the properties specified in {@code creationProperties}.
+     * @param streamId the unique identification of a persistent stream
+     * @param creationProperties properties to initialize the persistent stream if it does not exist yet
+     * @return a PersistedStream streaming events per segment
+     */
+    PersistentStream openPersistentStream(String streamId, PersistedStreamProperties creationProperties);
+
+
+    /**
+     * Deletes a persistent stream.
```

Comment:
Perhaps explicitly mention that this command will also return normally if the stream does not exist (if that's the way it actually behaves, of course)